front_end/helper.py

```diff
@@ -67,6 +68,9 @@ def convert_markdown_to_html(text):
     markdown = Markdown()
 
```

Comment:
Since the markdown module we're using wasn't registering \`\`\` marks, I used regex to substitute manually.